src/Components/App/App.test.js

```diff
@@ -48,7 +48,7 @@ describe('App', () => {
     const statsIcon = getByAltText('stats icon')
     fireEvent.click(statsIcon)
 
-    const statsHeading = getByRole('heading', { name: /⏳ stats page coming soon! ⏳/i })
+    const statsHeading = getByRole('heading', { name: /stats page coming soon!/i })
```

Comment:
Glad we were able to get the tests passing again without too much trouble. That was certainly an adventure!

Reply:
Twas an adventure indeed!!! Thanks for all the help 🙏🏼

Reply:
We <3 version control!!!